tests/Unit/Emogrifier/HtmlProcessor/CssInlinerTest.php

```diff
@@ -1223,6 +1223,30 @@ public function emogrifyByDefaultIgnoresInvalidCssSelectors()
         static::assertContains('background-color: blue', $html);
     }
 
+    /**
+     * @param string[][] $datasetsWithSpaces
+     *
+     * @return string[][] Datasets with spaces in the first array entry of each replaced with various alternative
+     *                    whitespace.  Keys will have " with ..." appended to provide an appropriate description.
+     */
+    private function getDatasetsWithVaryingWhitespace(array $datasetsWithSpaces)
+    {
+        $spaceReplacements = [
+            'extra spaces' => '  ',
```

Comment:
It is intentional that these are two spaces instead of one? This is inconsistent with the other two elements that have only white newline/Windows line ending.